com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollStageAppStatusExecution.java

```diff
@@ -45,7 +45,7 @@ public AsyncExecutionState execute(ExecutionWrapper execution) {
                 .debug(Messages.CHECKING_APP_STATUS, app.getName());
 
             StagingState state = applicationStager.getStagingState(execution, client);
-
+            LOGGER.info("Polled staging state: {} for app: {}", state.getState(), app.getName());
```

Comment:
In that way, the logs of the deploy-service in kibana will be a lot and some logs could be dropped.
If we really need this, we could do it in the catch blocks where the exception is risen.